pages/services/prometheus/0.1.0-2.3.1/command-reference/index.md

```diff
@@ -0,0 +1,227 @@
+---
+layout: layout.pug
+navigationTitle:  Command Reference
+title: Command Reference
+menuWeight: 60
+excerpt: Commands used in the DC/OS Prometheus Service
+featureMaturity:
+enterprise: false
+---
+
+
+usage: dcos prometheus
+  
+
+
+Flags:
+
+  -h, --help               Show context-sensitive help.
+  
+  -v, --verbose            Enable extra logging of requests/responses
+  
+      --name="prometheus"  Name of the service instance to query
+
+Commands:
```

Comment:
looks like the actual commands are missing

Reply:
Updated , let us know if you are looking for something else , we copied this command from cli